### ISSUE TITLE
create a quick start guide

### ISSUE THREAD

**Issue by JannisBush** [babe-project/babe_site#30]
The necessary documentation to realize an experiment is already quite long (sections: experiments, server app, data collection) and is likely to become even longer. 

It would be nice to have a quick start guide with just the minimal amount of information and pointers to the documentation.

**Comment by michael-franke** [comment]
yes! originally, the plan was to have a ‘gettting started’ section and a ‘full docs’ section on the _babe site.
